src/test/README.md

```diff
@@ -100,8 +101,9 @@ Run `test_bitcoin --help` for the full list of tests.
 
 ### Adding test cases
 
-To add a new unit test file to our test suite you need
-to add the file to `src/Makefile.test.include`. The pattern is to create
+To add a new unit test file to our test suite, you need
+to add the file to either `src/test/CMakeLists.txt` or
```

Comment:
I see you've mentioned the wallet as well, should we maybe mention that test utility functions could be extracted to `src/test/util/CMakeLists.txt` (or is that too advanced for an intro readme)?

Reply:
> or is that too advanced for an intro readme

It seems so.